src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/prometheus/monitoring-list/monitoring-list.component.html

```diff
@@ -0,0 +1,38 @@
+<tabset #tabs>
+  <tab id="active-alerts"
+       heading="Active Alerts"
+       (selectTab)="setFragment($event)"
+       i18n-heading>
+    <cd-active-alert-list *ngIf="isAlertmanagerConfigured"></cd-active-alert-list>
+    <cd-alert-panel type="warning"
+                    i18n
+                    *ngIf="!isAlertmanagerConfigured">To see all active Prometheus alerts, please
+      provide the URL to the API of Prometheus' Alertmanager as described in the
+  <a href="{{docsUrl}}"
+     target="_blank">documentation</a>.</cd-alert-panel>
+  </tab>
+  <tab id="all-alerts"
+       heading="All Alerts"
+       (selectTab)="setFragment($event)"
+       i18n-heading>
+    <cd-rules-list *ngIf="isPrometheusConfigured"
+                   [data]="prometheusAlertService.rules"></cd-rules-list>
+    <cd-alert-panel type="warning"
+                    *ngIf="!isPrometheusConfigured">To see all Prometheus configured alerts, please provide the URL to
```

Comment:
```suggestion
                    *ngIf="!isPrometheusConfigured">To see all configured Prometheus alerts, please provide the URL to
```
or
```suggestion
                    *ngIf="!isPrometheusConfigured">To see all in Prometheus configured alerts, please provide the URL to
```

Reply:
Actually, the proper wording should be "To see all alerts configured in Prometheus..."